src/core/symbology-ng/qgsstyle.h

```diff
@@ -273,6 +273,9 @@ class CORE_EXPORT QgsStyle : public QObject
     //! Changes ramp's name
     bool renameColorRamp( const QString& oldName, const QString& newName );
 
+    //! Creates a temporary memory database
+    bool createMemoryDB();
```

Comment:
Actually, ignore that. It does look useful. I'd just expand on the docs to explain why you'd want to call this and what effect it has (something like "if you don't want to associate the style with an on-disk database, ...etc". I'd also make a note in the openDb docs.